hack/e2e.sh

```diff
@@ -259,13 +277,24 @@ for ((i = 1; i <= 32; i++)) {
 EOF
     done
     echo "info: installing local-volume-provisioner"
-    $KUBECTL_BIN apply -f ${ROOT}/manifests/local-dind/local-volume-provisioner.yaml
+    $KUBECTL_BIN --context $KUBECONTEXT apply -f ${ROOT}/manifests/local-dind/local-volume-provisioner.yaml
     e2e::__wait_for_ds kube-system local-volume-provisioner
 }
 
 function e2e::setup_helm_server() {
-    $KUBECTL_BIN apply -f ${ROOT}/manifests/tiller-rbac.yaml
-    $HELM_BIN init --service-account=tiller --wait
+    $KUBECTL_BIN --context $KUBECONTEXT apply -f ${ROOT}/manifests/tiller-rbac.yaml
+    if hack::version_ge $KUBE_VERSION "v1.16.0"; then
+        # workaround for https://github.com/helm/helm/issues/6374
+        # TODO remove this when we can upgrade to helm 2.15+, see https://github.com/helm/helm/pull/6462
+        $HELM_BIN init --service-account tiller --output yaml \
+            | sed 's@apiVersion: extensions/v1beta1@apiVersion: apps/v1@' \
+            | sed 's@  replicas: 1@  replicas: 1\n  selector: {"matchLabels": {"app": "helm", "name": "tiller"}}@' \
+            | $KUBECTL_BIN --context $KUBECONTEXT apply -f -
+        echo "info: wait for tiller to be ready"
+		e2e::__wait_for_deploy kube-system tiller-deploy
```

Comment:
we need this hack because helm (<= 2.14) cannot be installed in 1.16.0+ via  `helm init` command